.github/workflows/bikesharing.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
     - master
     paths:
-    - samples/BikeSharingApp/**
+    - samples/BikeSharingApp/**/*
```

Comment:
>**/* [](start = 29, length = 4)

I'm just curious about the difference this makes compared to the previous version: previously, we were only listening to changes right under BikeSharingApp, whereas now we go also in deeper levels?

Reply:
Yep, recursive now